backend/src/test/java/gov/cdc/usds/simplereport/service/PersonServiceTest.java

```diff
@@ -654,38 +691,48 @@ void getPatients_search_OrgAdminUser() {
 
     // delete some data to verify archived works as expected
     // delete Charles (_site1)
-    Person charles = _service.getPatients(null, 0, 5, false, null, false).get(0);
+    Person charles =
+        _service.getPatients(null, 0, 5, ArchivedStatus.UNARCHIVED, null, false).get(0);
     _service.setIsDeleted(charles.getInternalId(), true);
     // Delete Frank (_site2)
-    Person frank = _service.getPatients(site2Id, 0, 5, false, null, false).get(0);
+    Person frank =
+        _service.getPatients(site2Id, 0, 5, ArchivedStatus.UNARCHIVED, null, false).get(0);
     _service.setIsDeleted(frank.getInternalId(), true);
 
     // all facilities, not deleted, "ma"
-    List<Person> patients = _service.getPatients(null, 0, 100, false, "ma", false);
+    List<Person> patients =
+        _service.getPatients(null, 0, 100, ArchivedStatus.UNARCHIVED, "ma", false);
     assertPatientList(patients, GALE, JANNELLE, KACEY, ELIZABETH, HEINRICK);
+    // all facilities, deleted
+    List<Person> deletedPatients =
+        _service.getPatients(null, 0, 100, ArchivedStatus.ARCHIVED, null, false);
+    assertPatientList(deletedPatients, CHARLES, FRANK);
 
     // site2, not deleted, "ma"
-    patients = _service.getPatients(site2Id, 0, 100, false, "ma", false);
+    patients = _service.getPatients(site2Id, 0, 100, ArchivedStatus.UNARCHIVED, "ma", false);
     assertPatientList(patients, JANNELLE, KACEY);
+    // site2, deleted, "fran"
+    deletedPatients = _service.getPatients(site2Id, 0, 100, ArchivedStatus.ARCHIVED, "fran", false);
```

Comment:
@fzhao99 -- testing patients for "ARCHIVED" status here from a specific facility with text "fran"